enterprise/server/index.md

```diff
@@ -144,13 +153,17 @@ If you've obtained Tomcat from your distribution's package manager, refer to you
 
 Create or edit the script `/opt/tomcat/bin/setenv.sh` to contain the following line:
 
-    CATALINA_OPTS=" -Dephox.config.file=/config/file/location/application.conf"
+```
+CATALINA_OPTS=" -Dephox.config.file=/config/file/location/application.conf"
```

Comment:
please confirm  -Dephox.config is correct, theres a few of them

Reply:
Nope. this is correct.